x/stake/keeper/delegation.go

```diff
@@ -567,7 +575,6 @@ func (k Keeper) BeginRedelegation(ctx sdk.Context, delAddr sdk.AccAddress,
 	if err != nil {
 		return types.Redelegation{}, err
 	}
-	k.OnDelegationCreated(ctx, delAddr, valDstAddr)
```

Comment:
k - so this change is fine because `OnDelegationModified` called within delegate for the new delegation. ++